simple/stats/constants.py

```diff
@@ -26,6 +26,22 @@
 INPUT_COLUMN_DCID = "dcid"
 INPUT_COLUMN_GEO_ID = "geoId"
 INPUT_COLUMN_LAT_LNG = "lat#lng"
+"""
+Dictionary of columns names that are considered to be already resolved 
+# to prefixes that the corresponding values should be prefixed with before the CSV is imported.
+
+e.g. if a column is named "geoId" and the values of that column in the CSV are: 01, 02; 
+then the importer will update the values to geoId/01, geoId/02 before importing them.
+
+Note that the lookup into these columns will always be lower-cased so ensure that 
+the keys in the dictionary are lower-case as well.
+
+Also, keep the keys sorted so it's easier to spot check should the list get large.
+"""
+PRE_RESOLVED_INPUT_COLUMNS_TO_PREFIXES = dict([("dcid", ""),
```

Comment:
for wikidataId, we still want to resolve them to be dcid using the resolve api: 
![image](https://github.com/datacommonsorg/import/assets/5951856/b0aef82b-f9aa-4490-86ae-031297b431f5)